src/app/data-explorer/data-schema-row/data-schema-row.component.html

```diff
@@ -0,0 +1,75 @@
+<!--
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div *ngIf="dataSchema" class="highlight-box mdm-data-schema-row">
+  <div class="mdm-data-schema-row__header">
+    <span class="mdm-data-schema-row__header-left-container">
+      <div class="mdm-data-schema-row__header-title">
+        <mat-checkbox
+          class="mdm-data-schema-row__header-title-checkbox"
+          [checked]="dataSchema.schema.isSelected"
+          (change)="schemaChecked($event)"
+        ></mat-checkbox>
+        <h3>{{ dataSchema.schema.label }}</h3>
+      </div>
+    </span>
+    <span class="mdm-data-schema-row__header-button-panel">
+      <button
+        *ngIf="canDelete"
+        mat-raised-button
+        class="mdm-data-schema-row__header-button fa fa-trash-alt"
+        color="primary"
+        (click)="removeSchema()"
+      ></button>
+
+      <mdm-data-element-in-request
+        class="mdm-data-schema-row__header-button"
+        *ngIf="sourceTargetIntersections.dataAccessRequests.length > 0"
+        [dataElements]="schemaElements"
+        [sourceTargetIntersections]="sourceTargetIntersections"
+        [caption]="'Add to other requests'"
+        [suppressViewRequestsDialogButton]="suppressViewRequestsDialogButton"
+        (requestAddDelete)="handleRequestAddDelete($event)"
+      ></mdm-data-element-in-request
+      ><!--(createRequestClicked)="handleCreateRequest($event)"-->
+      <button
+        mat-button
+        class="mdm-data-schema-row__header-button-updown fa-solid"
+        [ngClass]="{ 'fa-angle-up': visible, 'fa-angle-down': !visible }"
+        (click)="toggleCollapse()"
+      ></button>
+    </span>
+  </div>
+  <div [hidden]="!visible" class="mdm-my-request__request-elements" cCollapse>
```

Comment:
What is the `cCollapse` attribute for? A typo?

Reply:
This is a hangover from my initial attempts to expand and collapse sections. Deleted.